scripts/build/termux_step_massage.sh

```diff
@@ -197,7 +197,7 @@ termux_step_massage() {
 		local LIBOMP_A=$(${TERMUX_HOST_PLATFORM}-clang -print-file-name=libomp.a)
 		[[ "${LIBOMP_SO}" == "libomp.so" ]] && echo "WARN: LIBOMP_SO=${LIBOMP_SO}, discarding" >&2 && LIBOMP_SO=""
 		[[ "${LIBOMP_A}" == "libomp.a" ]] && echo "WARN: LIBOMP_A=${LIBOMP_A}, discarding" >&2 && LIBOMP_A=""
-		local LIBOMP_SO_SYMBOLS LIBOMP_A_SYMBOLS LIBOMP_SYMBOLS
+		local LIBOMP_SO_SYMBOLS=LIBOMP_A_SYMBOLS LIBOMP_SYMBOLS
 		[[ -n "${LIBOMP_SO}" ]] && LIBOMP_SO_SYMBOLS=$(${READELF} -s "${LIBOMP_SO}" | grep -E "GLOBAL[[:space:]]+DEFAULT" | grep -vE "[[:space:]]UND[[:space:]]" | grep -vE "[[:space:]]sizes$" | awk '{ print $8 }')
```

Comment:
The failure case described in #21155 should only be able to occur if either:
- `$LIBOMP_SO` is not empty, so this line gets skipped.
- Or this pipeline returns no matches, causing `$LIBOMP_SO_SYMBOLS` to be empty.